src/main/java/edu/cornell/kfs/module/purap/document/IWantDocument.java

```diff
@@ -100,6 +100,8 @@ public class IWantDocument extends FinancialSystemTransactionalDocumentBase impl
     private boolean completed;
     private String processorNetId;
     private String processorName;
+    private String purchasingAssistantNetId;
+    private String purchasingAssistantName;
```

Comment:
should this be transient since it isn't persisted?

Reply:
updated